src/clinical/service-worker-thread/tasks.ts

```diff
@@ -180,8 +180,10 @@ const mapEntityDocuments = (
 	switch (sortType) {
 		case ClinicalDataSortTypes.defaultDonor:
 			records = results.sort(sortDocs(sort, completionStats, sortDonorRecordsByCompletion));
+			break;
 		case ClinicalDataSortTypes.invalidEntity:
 			records = sortInvalidRecords(errors, results, entityName);
+			break;
 		case ClinicalDataSortTypes.columnSort:
 		default:
 			const sortKey = sort[0] === '-' ? sort.split('-')[1] : sort;
```

Comment:
Appreciate you adding the break statements.

Other things that I would reccommend are:
- use brackets to define a scoped block for each case
- add a comment after the `columnSort` case stating that you are intentionally falling through to default

Reply:
That's fine. These are stylistic things not functional.